tests/features/bootstrap/FeatureContext.php

```diff
@@ -170,6 +399,98 @@ public function assertFieldOptionNotSelected($option, $select) {
   }
 
   /**
+   * Checks if a node of a certain type with a given title exists.
+   *
+   * @param string $type
+   *   The node type.
+   * @param string $title
+   *   The title of the node.
+   *
+   * @Then I should have a :type (content )page titled :title
+   */
+  public function assertContentPageByTitle($type, $title) {
+    $type = $this->getEntityByLabel('node_type', $type);
+    // If the node doesn't exist, the exception will be thrown here.
+    $this->getEntityByLabel('node', $title, $type->id());
+  }
+
+  /**
+   * Checks the users existence.
+   *
+   * @param string $username
+   *   The username of the user.
+   *
+   * @throws \Exception
+   *   Thrown when the user is not found.
+   *
+   * @Then I should have a :username user
+   */
+  public function assertUserExistence($username) {
+    $user = user_load_by_name($username);
+
+    if (empty($user)) {
+      throw new \Exception("Unable to load expected user " . $username);
+    }
+  }
+
+  /**
+   * Click on an element by css class.
+   *
+   * @Then /^I click on element "([^"]*)"$/
+   */
+  public function iClickOn($element) {
+    $page = $this->getSession()->getPage();
+    $findName = $page->find('css', $element);
+    if (!$findName) {
+      throw new \Exception($element . " could not be found");
+    }
+    else {
+      $findName->click();
+    }
+  }
+
+  /**
+   * Resolves contextual links directly, without the need for javascript.
+   *
+   * @Then I click the contextual link :linkText in the :region region
+   */
+  public function iClickTheContextualLinkInTheRegion($linkText, $region) {
```

Comment:
The [coding standards](https://www.drupal.org/coding-standards#naming) actually allow both now, but it's correct that we should stick to one naming convention only. Again this was just moved so I didn't check it.